python/tests/platform/test_shared_pipeline.py

```diff
@@ -19,6 +19,7 @@ def test_create_pipeline(self):
         """
         CREATE TABLE tbl(id INT) WITH ('materialized' = 'true');
         CREATE MATERIALIZED VIEW v0 AS SELECT * FROM tbl;
+        CREATE MATERIALIZED VIEW "V0" AS SELECT * FROM tbl WHERE id % 2 <> 0;
```

Comment:
another case where this is necessary is when the names match existing keywords, like "date".

Reply:
Yes, and it does work, those keywords can only be defined with quotes, and are case sensitive. I added it to the test.